recipes/net-setSimultaneousAccepts-migration/src/workflow.ts

```diff
@@ -0,0 +1,309 @@
+	edits: Edit[]
+): void {
+	const bindingPath = resolveBindingPath(statementNode, '$');
+	if (!bindingPath) return;
+
+	const callExpressions = findSetSimultaneousAcceptsCalls(rootNode, bindingPath);
+
+	for (const callNode of callExpressions) {
+		const argNode = callNode.getMatch('ARG');
+		
+		if (argNode) {
+			handleCallArgument(rootNode, argNode, linesToRemove);
+		}
+
+		removeCallExpression(callNode, linesToRemove, edits);
+	}
+}
+
+/**
+ * Finds all _setSimultaneousAccepts() call expressions
+ */
+function findSetSimultaneousAcceptsCalls(
+	rootNode: SgNode<Js>,
+	bindingPath: string
+): SgNode<Js>[] {
+	return rootNode.findAll({
+		rule: {
+			pattern: `${bindingPath}._setSimultaneousAccepts($ARG)`,
+		},
+	});
+}
+
+/**
+ * Handles the argument passed to _setSimultaneousAccepts()
+ * If it's a member expression or identifier that's only used here, marks it for removal
+ */
+function handleCallArgument(
+	rootNode: SgNode<Js>,
+	argNode: SgNode<Js>,
+	linesToRemove: Range[]
+): void {
+	const argKind = argNode.kind();
+
+	if (argKind === 'member_expression') {
+		handleMemberExpressionArgument(rootNode, argNode, linesToRemove);
+	} else if (argKind === 'identifier') {
+		handleIdentifierArgument(rootNode, argNode, linesToRemove);
+	}
+}
+
+/**
+ * Handles member expression arguments (e.g., config.flag)
+ * Removes the property from the object if it's only used in this call
+ */
+function handleMemberExpressionArgument(
+	rootNode: SgNode<Js>,
+	argNode: SgNode<Js>,
+	linesToRemove: Range[]
+): void {
+	const objectNode = argNode.child(0);
+	const propertyNode = argNode.child(2);
+	
+	if (!objectNode || !propertyNode) return;
+
+	const objectName = objectNode.text();
+	const propertyName = propertyNode.text();
+	
+	const propertyRefs = rootNode.findAll({
+		rule: { pattern: `${objectName}.${propertyName}` },
+	});
+	
+	// Only remove if this is the only reference
+	if (propertyRefs.length === 1) {
+		removePropertyFromObjectDeclaration(rootNode, objectName, propertyName, linesToRemove);
+	}
+}
+
+/**
+ * Removes a property from an object literal declaration
+ */
+function removePropertyFromObjectDeclaration(
+	rootNode: SgNode<Js>,
+	objectName: string,
+	propertyName: string,
+	linesToRemove: Range[]
+): void {
+	const objDeclarations = rootNode.findAll({
+		rule: {
+			any: [
+				{ pattern: `const ${objectName} = $_` },
+				{ pattern: `let ${objectName} = $_` },
+				{ pattern: `var ${objectName} = $_` },
+			],
+		},
+	});
+	
+	for (const objDecl of objDeclarations) {
+		const objectLiterals = objDecl.findAll({ rule: { kind: 'object' } });
+		
+		for (const obj of objectLiterals) {
+			const pairs = obj.findAll({ rule: { kind: 'pair' } });
+			
+			for (const pair of pairs) {
+				const key = pair.child(0);
+				if (key?.text() === propertyName) {
+					const rangeWithComma = expandRangeToIncludeTrailingComma(
+						pair.range(),
+						rootNode.text()
+					);
+					linesToRemove.push(rangeWithComma);
+				}
+			}
+		}
+	}
```

Comment:
can we have a loop that push to an array then iterate on this array to avoid 3th level of nested loop